### ISSUE TITLE
Always get unexpected error while trying `prisma2 dev`

### ISSUE THREAD

**Issue by deadcoder0904** [prisma/prisma2#639]
I was trying Prisma Studio for the 1st time. I typed `prisma2 dev` in terminal & keep getting this

![Error Prisma Studio](https://user-images.githubusercontent.com/16436270/65831866-7f1f8c80-e2db-11e9-8ad2-e6f55d9bc5af.png)

My Prisma version is latest preview-13

**Comment by janpio** [comment]
Thanks for reporting this @deadcoder0904, we are aware and working on fixing this right now (assuming you are hitting the same one). There should be a hotfix out soon-ish - the straight forward way here is for you to then try again and let us know if this fixed the issue for you.

**Comment by timsuchanek** [comment]
We just updated the error prompt in `preview-13.1` to also include the stack trace.
The concrete problem is very likely a breaking change in the migrations table, which Lift creates in your database.
You can read more about the breaking change here https://github.com/prisma/prisma2/releases/tag/2.0.0-preview-13

**Comment by timsuchanek** [comment]
Please let us know, if it's the `missing field \`original_steps\`` error.

**Comment by deadcoder0904** [comment]
@timsuchanek I get the same error as seen in the screenshot on the release of 13.1.

```bash
ERROR  Oops, an unexpected error occured!
Error in migration engine: thread 'tokio-runtime-worker-0' panicked
 at 'Deserializing the database migration failed.: Error("missing f
ield `original_steps`", line: 0, column: 0)', src/libcore/result.rs
:999:5
```

![Screenshot](https://user-images.githubusercontent.com/1094804/65869684-33242480-e37b-11e9-8f93-4abe6ec0425d.png)

**Comment by deadcoder0904** [comment]
And when I submit "Yes" then the screen keeps on jumping for a while (~5secs). Check GIF below 👇

![error](https://user-images.githubusercontent.com/16436270/65871098-56f76280-e39b-11e9-89b9-0b1e58538a08.gif)

**Comment by timsuchanek** [comment]
Thanks for the info. We'll look into the ugly jumping while it submits the error.
Could you resolve the error by deleting the `_Migration` table from you database?
You can e.g. you Table Plus to do so.

**Comment by deadcoder0904** [comment]
I deleted the `_Migration` database but it still gives me the same error. Did I had to delete only table?

**Comment by janpio** [comment]
From my understanding it is the `_Migration` _table_ to be deleted.

**Comment by divyenduz** [comment]
@deadcoder0904 You had to delete the `_Migration` table. I am not sure how you ended up with a `_Migration` database? 🤔

**Comment by deadcoder0904** [comment]
@divyenduz Ohh sorry, I am not attentive today lmao.

I deleted `_Migration` table only. I don't think there was a `_Migration` database. Silly me. But it still doesn't work. I still have another table with the data.

Also, I used [SQL Pro for SQLite](https://www.sqlitepro.com/) for deleting it :)

**Comment by divyenduz** [comment]
What error do you get with preview 13.1? Can you please share a sceenshot?

**Comment by deadcoder0904** [comment]
I guess it's the same error but have a look at it urself. The error was big so included Part 1 & 2 :)

# Part 1 

![Part 1](https://user-images.githubusercontent.com/16436270/65880262-21f50b00-e3af-11e9-9a91-6c1d7b941b2d.png)

# Part 2

![Part 2](https://user-images.githubusercontent.com/16436270/65880386-5b2d7b00-e3af-11e9-82bc-320376a80e87.png)

**Comment by divyenduz** [comment]
There is a type in your schema that we don't support, would it be possible for you to share your schema file?

**Comment by deadcoder0904** [comment]
This is all I have 👇

```js
model Blog {
  id             String   @default(cuid()) @id
  createdAt      DateTime @default(now())
  updatedAt      DateTime @updatedAt
  title          String
  link           String?
}
```

**Comment by divyenduz** [comment]
I can't repro with SQLite, maybe you want to remove the whole DB and start over? Or send your project to me as a zip file for me to investigate further?

**Comment by deadcoder0904** [comment]
So I started over by dropping the DB & seeding DB again. Went to http://localhost:5555/ & got the following error in `Console`

![Prisma Studio Console](https://user-images.githubusercontent.com/16436270/65888949-06ddc780-e3be-11e9-9b52-a7bf61d752e0.png)

**Comment by deadcoder0904** [comment]
One weird thing is `Ctrl+C` doesn't stop the script in the terminal while generating `Photon.js Client`, `Nexus`, etc... which feels unintuitive.

It also asks me `To save changes into a migration file, run prisma2 lift save` but when I enter this in the terminal, I get the following:

```bash
➜ prisma2 lift save
Everything up-to-date
```

**Comment by deadcoder0904** [comment]
There's also a lot of blank space above `Studio endpoint: http://localhost:5555/`

![prisma2 dev](https://user-images.githubusercontent.com/16436270/65889598-26c1bb00-e3bf-11e9-89e5-ae9885e34af4.png)

**Comment by divyenduz** [comment]
@deadcoder0904 That is a separate issue, please create an issue here: https://github.com/prisma/studio

**Comment by janpio** [comment]
> There's also a lot of blank space above `Studio endpoint: http://localhost:5555/`

That is by design here, as this should be a full screen interactive view. (Still good if you create an issue that this confused you over at prisma/studio)

**Comment by deadcoder0904** [comment]
> @deadcoder0904 That is a separate issue, please create an issue here: https://github.com/prisma/studio

@divyenduz  I opened 3 separate issues here → https://github.com/prisma/studio/issues/182, https://github.com/prisma/studio/issues/183 & https://github.com/prisma/studio/issues/184

> That is by design here, as this should be a full screen interactive view. (Still good if you create an issue that this confused you over at prisma/studio)

@janpio I explained why it affected me here → https://github.com/prisma/studio/issues/184#issue-500383752

TL;DR i was using terminal with height small so I couldnt see the endpoint. wasted 15 mins trying to find endpoint url then expanded my terminal to find it. maybe put the endpoint above everything else. after all, its the most important part of studio :)

**Comment by lednhatkhanh** [comment]
Got the exact same error: `missing field `original_steps`` when trying to run migration with `lift` and `prisma2@2.0.0-preview013.3`.

**Comment by pantharshit00** [comment]
Hi @lednhatkhanh 

Can you also make sure that you have deleted the `_Migration` table and run `prisma2 dev`

**Comment by divyenduz** [comment]
I am closing this one due to inactivity, if you still think that this is an issue, please comment here and tag me. Thank you 🙏
